apps/ledger-live-desktop/src/renderer/components/Onboarding/Screens/Tutorial/index.tsx

```diff
@@ -700,7 +702,10 @@ export default function Tutorial({ useCase }: Props) {
     [history],
   );
 
-  const handleNextPin = useCallback(() => {
+  const recoverServices = useFeature("protectServicesDesktop");
+  const recoverStoreId = recoverServices?.params?.protectId ?? "";
+
+  const handleNextPin = useCallback(async () => {
```

Comment:
Can be removed after removing the `await` below
```suggestion
  const handleNextPin = useCallback(() => {
```

Reply:
We can still remove this I think